backend/minors/POR.yaml

```diff
@@ -0,0 +1,89 @@
+---
+type: Minor
+name: Portuguese
+code: POR
+degree:
+urls:
+- https://spo.princeton.edu/undergraduate/minor
+description: |- 
+  Students wishing to complete a minor in the Department of Spanish and 
+  Portuguese must complete the Minor Application Form. 
+
+  Applications may be submitted as early as the second semester of the 
+  sophomore year, after a major has been declared. We highly recommend 
+  students apply by the Spring term of their junior year to ensure 
+  course plans can be built accordingly. Applications will be accepted 
+  no later than February 15 of the senior year.
+
+  Students who want to minor in both Spanish or Portuguese and the 
+  Program in Latin American Studies can do so with the permission of 
+  both program directors, which includes a conversation about course 
+  plans. In agreement with the Program in Latin American Studies 
+  (PLAS), one course can be used towards both a minor in SPA or POR and 
+  a minor in PLAS.
+excluded_major: 
+- SPA
+contacts:
+- type: Director of Undergraduate Studies
+  name: Christina Lee
+  email: chrislee@princeton.edu
+- type: Graduate and Undergraduate Program Administrator
+  name: Janeth "Jane" Marquez
+  email: janeth.marquez@princeton.edu
+- type: Acting Director, Portuguese Language Program
+  name: Luis Gonçalves
+  email: lgoncalv@princeton.edu
+req_list:
+- name: Portuguese Courses
+  max_counted: 1
+  min_needed: 5
+  explanation: |-
+    Five POR or cross-listed courses at the 200-level or higher (with 
+    at least one at the 300-level or higher).
+
+    Princeton summer abroad courses in Portugal/Cape Verde or Brazil 
+    offered by SPO count towards the requirement.
+    
+    Up to one course per semester taken abroad at another institution 
+    can count towards the minor with the pre-approval of the DUS. 
+    Students intending to study abroad must contact the DUS with the 
+    description and syllabus of the Portuguese course they intend to 
+    have counted towards the minor to make sure that such course is 
+    equivalent to a course offered at Princeton.
+
+    Up to one cross-listed course taken in English can count towards 
+    the minor, provided that all written work is completed in 
+    Portuguese.
+  req_list:
+  - name: 100-Level Course
```

Comment:
Check out the Portuguese certificate [YAML](https://github.com/HoagieClub/plan/blob/5286f5433194b8663957c91b62c698ee07528132/backend/certificates/LAC-POR.yaml) for reference. I think we should add that to encode info about the 200-level courses